rita_common/src/payment_controller/mod.rs

```diff
@@ -252,8 +271,51 @@ async fn make_payment(mut pmt: PaymentTx) -> Result<(), PaymentControllerError>
         attempt: 0u8,
     };
 
-    match neigh_ack {
-        Ok(mut val) => {
+    // Hit both endpoints, in the case of a node having both endpoints, validator will simply get a duplicate transaction txid and discard it.
+    let (neigh_ack_v2, neigh_ack) = join(neigh_ack_v2, neigh_ack).await;
+
+    match (neigh_ack_v2, neigh_ack) {
+        (Ok(mut val2), Ok(mut val)) => {
+            // THis is probably a b20 router
+            match (val2.status().is_success(), val.status().is_success()) {
+                (true, _) => {
+                    info!(
+                        "Payment pmt with txid: {:#066x} is sent to our neighbor with status {:?} and body {:?} via url {}, using full node {} and amount {:?}",
+                        tx_id,
+                        val2.status(),
+                        val2.body().await,
+                        neighbor_url_v2,
+                        full_node,
+                        pmt.amount
+                    );
+                }
+                (false, true) => {
+                    error!(
+                        "Make_payment_v2 with txid {:#066x} failed with status {:?} and body {:?}",
+                        tx_id,
+                        val2.status(),
+                        val2.body().await
+                    );
+                    info!(
+                        "Payment pmt with txid: {:#066x} is sent to our neighbor with status {:?} and body {:?} via url {}, using full node {} and amount {:?}",
```

Comment:
can you use regular print not debug print for amount, it will make the numbers unusable to log parsers

Reply:
Fixed